package.sh

```diff
@@ -12,7 +12,7 @@ ignore_func_prefix="local"
 mkdir -p ${artefact_dir}
 functions=$(npx yaml2json serverless.yml | jq -r '.functions | keys | .[]')
 for func_name in ${functions}; do
-  if [ "$func_name"=="*$ignore_func_prefix*" ]; then
+  if [[ $func_name == *"$ignore_func_prefix"* ]]; then
```

Comment:
If we're doing it via prefix, shouldn't we omit the first `*`?

Reply:
I'd like to keep it in to catch all instances where 'local' appears in a function name, as most likely we shouldn't be packaging them. However, I did say 'prefix', so feel free to berate me for that